Makefile

```diff
@@ -98,7 +98,8 @@ test_unit: clean static_external
 
 # Check whether migrations need to be generated
 test_migrations_missing: clean
-	! honcho -e .env.test run ./manage.py makemigrations --dry-run --exit
+	(psql -lqt | cut -d \| -f 1 | grep -qw opencraft) || (createdb --encoding utf-8 --template template0 opencraft)
```

Comment:
@itsjeyd Any idea _why_ this is happening now, and didn't happen in the past?  It looks like this should always have been an issue.  I can't figure out by what mechanism the DB was created on CircleCI before this PR.

In any case, the make target `test_migrations_missing` seems like a weird place for the implicit creation of the database.  If this is indeed necessary, it can be simplified to

    createdb --encoding utf-8 --template template0 opencraft 2> /dev/null || true